nav2_waypoint_follower/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 cmake_minimum_required(VERSION 3.5)
 project(nav2_waypoint_follower)
 
+# Default to C++17
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
I think `nav2_common()` set to 14, so I think you should set it regardless.

Reply:
you mean `nav2_package`()

Reply:
yes, thanks for the correction